adapters/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys
+sys.path.append("../")
```

Comment:
instead of doing this in every dir, can we have a __iinit__.py at the root of the workspace, and do instead
sys.path.append(os.path.dirname(os.path.realpath(__file__))) ?

Reply:
Done.